### ISSUE TITLE
things to check

### ISSUE THREAD

**Issue by gisab** [i5Js/WD-EX2-Ultra#1]
Hi I5Js.
To my memory....
I connected also the 3.3V and disabled the USB serial "power" from the PC (it should be the jumper on the botton in the photo).

The soldier are a bit dirty... have you checked with a tester if the 4 pins of the serial are in shortcut (i.e. you tester found a continuity between any couple of the 4 pin) ?

**Comment by gisab** [comment]
I checked. right now. I connected only 3 wires as you did.
However i make the soldies on the front side of the board.
How did you managed to have the pin crossing the board?

**Comment by i5Js** [comment]
I have checked the 3.3v, with this wire connected, the board is not detected in the PC, as you said, without it, it's detected.

The board had the "holes" to make the solides from any side.

Checking with the tester now.

**Comment by gisab** [comment]
can you add a photo of the connected wires from nas to serial?

**Comment by gisab** [comment]
in this photo:
https://github.com/gisab/WDMC-Ex2/tree/main/1.SerialCable
TX means (nas send data and) that shall be conected on the serial RX

**Comment by i5Js** [comment]
Yes, I have the wires crossed. tx-rx 

without any wires connected, there is no continuity between the the 4 pins, but wit the wires connected to the uart, there is continuity between rx and gnd.

Uploading the photo...

**Comment by gisab** [comment]
pls, can you upload a jpg?

**Comment by i5Js** [comment]
Done, sorry :)

**Comment by gisab** [comment]
yep the connection is wrong
the two wires on the high  (brown and red) should be one pin higher

**Comment by gisab** [comment]
you understood the issue?

**Comment by i5Js** [comment]
no hahahaha :)

**Comment by i5Js** [comment]
I have uploaded another photo.

**Comment by gisab** [comment]
i   don't understand the purpose of this new photo
is it working or not?

**Comment by i5Js** [comment]
No, it's not working either, I mean, it's same, but on the other side.

**Comment by i5Js** [comment]
I have another uart board, that doesn't work either, but perhaps you want to see it.

new pic is coming

**Comment by gisab** [comment]
try to go sstep by step
just with GND and RX you should be able to see the blinking led during the boot phase

**Comment by i5Js** [comment]
ok, so start with RX and GND connected in the UART, right?

**Comment by gisab** [comment]
Yes!

**Comment by gisab** [comment]
Nas tx and uart rx
Gnd on both sides

**Comment by i5Js** [comment]
Done, and doesn't work :( I'm sorry man, I thing my board has the serial interface dead.

**Comment by gisab** [comment]
It could be

can i suggest last trial?
Plug the tester on nas and measure the V between Tx and GND, but directly on the board (try to avoid the pin you plugged). 
It should be 0V. 
Now reboot the nas. 
The TX should be up/down at 115000 baud.
It means that you should see an average V>0 (during the 1 min of boot time). 

If this is the case… either the uart is broken either the cabling.

**Comment by i5Js** [comment]
Ok, please understand, I don't have knowledge about testers or continuity. So, I've selected V...._____ 250 During the boot I can see -003. Selecting V~ is 0 all the time

**Comment by gisab** [comment]
Yep. Sound good
use V in continue “____” with a scale of 5V
Seeing flicking around 3V means the pin is working (the tester give an average value)
If you see a static 3.3V or 5V means the pin is always high, i.e. you connected to V either you solded a shortcut with the other V pin.

**Comment by i5Js** [comment]
with then nas off, but the UART connected to the PC, I am seeing 3.26 all the time.

**Comment by gisab** [comment]
Just for this test… disconnect the uart and plug the tester on the nas

**Comment by i5Js** [comment]
0.02-0.03 when it's booting, 0.00 when it is up

**Comment by gisab** [comment]
Yep. So the pin is working. 
when you connect the uart do you see 3.3v??
This is wrong. 
are you sure you did not connect to 3.3V?

**Comment by i5Js** [comment]
Double checking...

**Comment by i5Js** [comment]
Done, I'm sure, the two pins closer, 3.3v in the tester, all the time

**Comment by gisab** [comment]
I guess… is the uart broken

**Comment by i5Js** [comment]
nas right?

**Comment by gisab** [comment]
Are you still using the one in the photo of this morning?
Or you used another one?

**Comment by gisab** [comment]
No the pc/uart

**Comment by i5Js** [comment]
I have used both :) With the second is 3.3v all the time, with the first one 3.26v

**Comment by gisab** [comment]
Your uart shortcut the Tx pin to 3.3V and therefore you dont see any signal…
But… still I don’t understand…
If the uart pin is at 3.3V … then the led should be lighting…. And this is not the case (?!?!?)

**Comment by gisab** [comment]
I guess there is something wrong again in the cabling….

**Comment by i5Js** [comment]
indeed, is not the case :D

**Comment by gisab** [comment]
Can you make a new photo with the cable?

**Comment by i5Js** [comment]
mmmm what about the usb-c to usb-a converter?could it explain it?

**Comment by gisab** [comment]
I dont think so, but I cannot exclude

**Comment by i5Js** [comment]
making photos...

**Comment by gisab** [comment]
Do you have friends with a knowledge of electrical signal?
Being there could be simpler to debug
: )))

**Comment by i5Js** [comment]
hahahahah no, I'm not so lucky

**Comment by i5Js** [comment]
Done, testing1,2,3 that is the other, sorry, I'm alone to do only one pic

**Comment by gisab** [comment]
testing 1 is perfect.

BUT ARE YOU CONNECTING THE NAS TX WITH THE UART TX ???????????????????????????

**Comment by gisab** [comment]
The brown (nax TX) should be connected on UART RX 
: ))))))))))))

**Comment by gisab** [comment]
TX -> RX
Nas Trasmit and UART Receive : )

**Comment by i5Js** [comment]
Fixed, no output and same measures :( 3.3v

**Comment by gisab** [comment]
please send a photo

**Comment by gisab** [comment]
do you have whatsapp or telegram?
send me your accounr; i will contact you.
it could be simpler to chat

**Comment by i5Js** [comment]
Sure!! Give me some minutes:)

Enviado desde dispositivo móvil
________________________________
De: Gianluca Sabella ***@***.***>
Enviado: Sunday, November 6, 2022 2:56:20 PM
Para: i5Js/WD-EX2-Ultra ***@***.***>
Cc: i5Js ***@***.***>; Comment ***@***.***>
Asunto: Re: [i5Js/WD-EX2-Ultra] things to check (Issue #1)


do you have whatsapp or telegram?
send me your accounr; i will contact you.
it could be simpler to chat

**Comment by i5Js** [comment]
telegram: @i5Jss

**Comment by i5Js** [comment]
Thanks for all your time today :)

**Comment by i5Js** [comment]
I've been able to install Debian 9, but the u-boot partition is in read only :(

```
root@wdmc:~# mtd_debug info /dev/mtd0
mtd.type = MTD_NANDFLASH
mtd.flags = MTD_CAP_ROM
mtd.size = 5242880 (5M)
mtd.erasesize = 131072 (128K)
mtd.writesize = 2048 (2K)
mtd.oobsize = 64 
regions = 0
```
